nautobot_floor_plan/forms.py

```diff
@@ -37,6 +37,18 @@ class FloorPlanForm(NautobotModelForm):
         help_text="The first value to begin Y Axis at.",
         required=False,
     )
+    x_axis_step = forms.IntegerField(
+        label="X Axis Step",
+        help_text="A positive or negative integer, excluding zero",
+        required=False,
+        validators=[utils.validate_not_zero],
+    )
```

Comment:
Edge case: Floor plan add/edit form doesn't require any of the seed or step fields, but sending None (delete the prefilled values) in any of them causes IntegrityError as the model fields cannot be null. We should either mark the fields on the form as required or default them if None is sent in.

Reply:
Will set these to required-True to fix this issue.